src/screens/ChannelListScreenV2/IncognitoEmptyChat/index.tsx

```diff
@@ -0,0 +1,155 @@
+import {DISCOVERY_TAB_TOPICS, NavigationConstants} from '../../../utils/constants';
+
+const {width: widthScreen} = Dimensions.get('window');
+
+const MODE_FULL = 1;
+const MODE_BUTTON = 2;
+const MODE_HIDE = 3;
+
+const IncognitoEmptyChat = ({totalChannel}) => {
+  const {goToContactScreen} = useAnonymousChannelListScreenHook();
+  const navigation = useNavigation();
+
+  const getMode = () => {
+    let result = MODE_FULL;
+    if (totalChannel === 0) {
+      result = MODE_FULL;
+    }
+    if (totalChannel >= 3) {
+      result = MODE_HIDE;
+    }
+    if ((totalChannel > 0 && totalChannel < 3) || widthScreen < 380) {
+      result = MODE_BUTTON;
+    }
+
+    return result;
+  };
+
+  if (getMode() === MODE_HIDE) return null;
+
+  return (
+    <View style={styles.container} testID="IncognitoEmptyChatContainer">
+      {getMode() === MODE_FULL && (
+        <Image testID="IncognitoEmptyChatImage" source={EmptyIncognito} style={styles.image} />
+      )}
+      {getMode() === MODE_FULL && (
+        <Text testID="IncognitoEmptyChatText" style={styles.titleText}>
+          All your incognito activity, including chats, posts, comments and communities will appear
+          here.
+        </Text>
+      )}
+      <View style={styles.wrapper}>
+        {getMode() === MODE_FULL && <View testID="IncognitoEmptyChatLine" style={styles.line} />}
+        <TouchableOpacity style={styles.item} onPress={() => goToContactScreen({from: ANONYMOUS})}>
+          <View style={styles.itemContent}>
+            <Text style={styles.itemTitle}>Start an Incognito Chat</Text>
+            <Text style={styles.itemSubtitle}>Send an incognito message to friends.</Text>
+          </View>
+          <View>
+            <IconArrowRight width={8} height={12} fill={COLORS.gray400} />
+          </View>
+        </TouchableOpacity>
+        <TouchableOpacity
+          style={styles.item}
+          onPress={() =>
+            navigation.navigate(NavigationConstants.CREATE_POST_SCREEN, {followType: 'incognito'})
+          }>
+          <View style={styles.itemContent}>
+            <Text style={styles.itemTitle}>Create an Incognito Post</Text>
+            <Text style={styles.itemSubtitle}>Post and comment using an alias.</Text>
+          </View>
+          <View>
+            <IconArrowRight width={8} height={12} fill={COLORS.gray400} />
+          </View>
+        </TouchableOpacity>
+        <TouchableOpacity
+          style={styles.item}
+          onPress={() => navigation.navigate('DiscoveryScreen', {tab: DISCOVERY_TAB_TOPICS})}>
+          <View style={styles.itemContent}>
+            <Text style={styles.itemTitle}>Join Communities in Incognito Mode</Text>
+            <Text style={styles.itemSubtitle}>Safely join without anyone knowing.</Text>
+          </View>
+          <View>
+            <IconArrowRight width={8} height={12} fill={COLORS.gray400} />
+          </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These three blocks of code are very similar and could be abstracted into a separate component or function to reduce redundancy and improve maintainability. Consider creating a new component, for example `IncognitoOption`, which takes the title, subtitle, onPress function, and testID as props.

```jsx
const IncognitoOption = ({title, subtitle, onPress, testID}) => (
  <TouchableOpacity style={styles.item} onPress={onPress}>
    <View style={styles.itemContent}>
      <Text style={styles.itemTitle}>{title}</Text>
      <Text style={styles.itemSubtitle}>{subtitle}</Text>
    </View>
    <View>
      <IconArrowRight width={8} height={12} fill={COLORS.gray400} />
    </View>
  </TouchableOpacity>
);
```
Then use this component in your render method:
```jsx
<IncognitoOption 
  title="Start an Incognito Chat" 
  subtitle="Send an incognito message to friends." 
  onPress={() => goToContactScreen({from: ANONYMOUS})}
/>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->